sale_revert_done/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+Sale Revert Done
+================
+
+This module extends the functionality of sales to allow you to set a sales
+order done back to state 'Sale Order'.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to a sales order in state 'done'.
+#. Click the button 'Back to Sale'.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/167/9.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/purchase-workflow/issues>`_. In case of trouble, please
```

Comment:
That's right for the runbot link. But for the issues:
* https://github.com/OCA/purchase-workflow/issues: leads to the issues of that repo.
* https://github.com/OCA/167/issues: Leads to nowhere.

In adition, I tried to follow the template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst